### DISCUSSION TITLE
ramp_up_how_to_learn

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/labview_book#27] [Announcements]
# ramp_up_how_to_learn

学习 LabVIEW 的三种方式

https://lv.qizhen.xyz/ramp_up_how_to_learn

**Comment by jincheng-1** [comment]
电子发烧友中labview论坛

**Reply by ruanqizhen** [reply]
电子发烧友论坛我前一阵子还能用，现在也用不了了，因为我没有国内的手机。近几年国内言论管控越来越严格，很多原本能使用的论坛都开始禁止无法手机认证的用户使用了。GitHub 没有这个问题，但我很担心哪一天国内就再也连不上 GitHub 了。

**Reply by xql-12345** [reply]
已经很难访问了，断断续续

**Comment by Kindler1** [comment]
真实，可读性强

**Comment by Riching5** [comment]
下载的最新版本2022 Q3的labview，即时帮助文档变成跳转到官网英文帮助网页了，请问这个有办法可以解决吗？

**Reply by ruanqizhen** [reply]
我还没有安装过 2022 的版本，等下次安装的时候，我会试一下。官方网页上有个语言设置，改为中文，之后就都会显示中文了。

**Comment by Riching5** [comment]
妙啊，谢谢啦&nbsp;


雷秦
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年1月9日(星期一) 凌晨5:01
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [ruanqizhen/labview_book] ramp_up_how_to_learn (Discussion #27)





 
我还没有安装过 2022 的版本，等下次安装的时候，我会试一下。官方网页上有个语言设置，改为中文，之后就都会显示中文了。

**Comment by foraiurs** [comment]
我最近在看一些labview程序，程序框图中有些连线比较混乱，我想问一下有没有什么方法准确判断一条线是连接的模块哪一个点？

**Reply by ruanqizhen** [reply]
双击一条数据线，可以让它高亮显示，帮助你查看它连接的所有端点。

**Reply by foraiurs** [reply]
谢谢

**Comment by yuanxiuliang** [comment]
“与文本编程语言相比，LabVIEW 的代码管理很分享是一个劣势”误写了一个字”管理和分享“

**Reply by ruanqizhen** [reply]
已修改。多谢指正！

**Comment by Tim-US** [comment]
我是小白，book很有帮助

**Comment by editfund-3rd** [comment]
💌😍不错⭐

**Comment by Acratons** [comment]
书很不错，最近看到后面，把网页上的vi片段拖labview中试试，然后直接变成图片地址，下载下来后可以，但是我用的18版本T^T

**Comment by ruanqizhen** [comment]
之前博客下的留言：

杰
2008年07月20日 22:45 编辑
“我见过几个工程师在项目中遇到了难题，于是来报名参加LabVIEW的培训课程，以为上完课可以解决自己的问题。但实际上完全误解了培训课程的意义，培训课程是为了帮助那些想要系统学习LabVIEW知识的人，而不专注于任何一个具体问题。”

PLS Check the information at NI official website regarding the training program, you can understand why those engineers thought the training could help them to solve all the problems. For example:
经济、高质的专业训练课程均由知识丰富的应用工程师讲授，他们可以和您讨论应用程序的解决方案，帮助您的项目快速起步。
帮助您开发大型应用程序，并能轻松进行维护及重复使用.

回复
杰
2008年07月20日 22:38 编辑
哈哈，小阮这个分析的好玩。
我首先是目标驱动型，完成任务之后变成了探索型，现在是系统学习型，一个VI一个VI的看，用用满好玩的，呵呵。
不过LabVIEW涵盖的东西和应用范围过广，还是只能学习纯软件的东西，很多工具包如果不是确实要用，基本上不会碰的。

回复

**Comment by ruanqizhen** [comment]
之前博客上的相关文章，搬到这里：

2009年5月18日 · 阅读需 3 分钟
如果在学习过程中遇到问题，可以考虑通过互联网来寻求帮助。把相关的问题张贴在相关的论坛或高手博客上，便可得到别的的帮助。

#### 论坛

<table border="1" width="700" cellspacing="0" cellpadding="2"><tbody><tr><td valign="top" width="248">论坛网址</td><td valign="top" width="138">论坛名称</td><td valign="top" width="310">注释</td></tr><tr><td valign="top" width="250"><a href="http://forums.ni.com/">http://forums.ni.com/</a> <a title="http://forums.ni.com/ni/board?board.id=2170" href="http://forums.ni.com/ni/board?board.id=2170">http://forums.ni.com/ni/board?board.id=2170</a></td><td valign="top" width="138">NI官方论坛 NI官方中文论坛</td><td valign="top" width="308">这个论坛活跃程度很高，并且时常有NI的技术支持和研发工程师来回答问题。论坛有专门的中文讨论区，但如果自己英文尚可，最好是去英文讨论区提问。英文讨论区人气更旺，更容易获得帮助。</td></tr><tr><td valign="top" width="252"><a title="http://forums.lavag.org/" href="http://forums.lavag.org/">http://forums.lavag.org/</a></td><td valign="top" width="138">LAVA (LabVIEW Advanced Virtual Architects)</td><td valign="top" width="307">这是官方之外最大的LabVIEW社区，也是寻求帮助的好地方。该网站没有官方背景，讨论起来可以更自由些，但这里只能用英文讨论。</td></tr><tr><td valign="top" width="253"><a title="http://www.vihome.com.cn/bbs/" href="http://www.vihome.com.cn/bbs/">http://www.vihome.com.cn/bbs/</a></td><td valign="top" width="137">测量与测试技术交流社区</td><td valign="top" width="306">这是目前最活跃的中文LabVIEW论坛。</td></tr><tr><td valign="top" width="255">https://bbs.elecfans.com/zhuti_labview_1.html</td><td valign="top" width="137">电子发烧友论坛</td><td valign="top" width="305"></td></tr><tr><td valign="top" width="256"><a title="http://www.cpubbs.com/bbs/forum-4-1.html" href="http://www.cpubbs.com/bbs/forum-4-1.html">http://www.cpubbs.com/bbs/forum-4-1.html</a></td><td valign="top" width="137">CPUBBS论坛</td><td valign="top" width="304"></td></tr><tr><td valign="top" width="257"><a title="http://bbs.avrvi.com/thread.php?fid-8.html" href="http://bbs.avrvi.com/thread.php?fid-8.html">http://bbs.avrvi.com/thread.php?fid-8.html</a></td><td valign="top" width="137">AVR与虚拟仪器</td><td valign="top" width="303"></td></tr><tr><td valign="top" width="258"><a title="http://forum.simwe.com/forum-25-1.html" href="http://forum.simwe.com/forum-25-1.html">http://forum.simwe.com/forum-25-1.html</a></td><td valign="top" width="136">仿真科技论坛</td><td valign="top" width="303"></td></tr></tbody></table>

#### MSN和QQ群

<table border="1" width="700" cellspacing="0" cellpadding="2"><tbody><tr><td valign="top" width="237">群地址</td><td valign="top" width="139">群名</td><td valign="top" width="321">注释</td></tr><tr><td valign="top" width="238"></td><td valign="top" width="139"></td><td valign="top" width="320"></td></tr></tbody></table>

#### 博客

（注释内容引自[“虚拟仪器技术文摘”博客](http://blog.gsdzone.net/u/17/index.html)）

<table border="1" width="699" cellspacing="0" cellpadding="2"><tbody><tr><td valign="top" width="292">博客网址</td><td valign="top" width="79">博客名称</td><td valign="top" width="326">注释</td></tr><tr><td valign="top" width="293"><a href="http://ruanqizhen.spaces.live.com/">http://ruanqizhen.spaces.live.com/</a></td><td valign="top" width="79"></td><td valign="top" width="325"></td></tr><tr><td valign="top" width="294"><a href="http://labview7i.blog.sohu.com/">http://labview7i.blog.sohu.com/</a></td><td valign="top" width="79"></td><td valign="top" width="325">第一次查有关LabVIEW的博客就查到了这里，以前是在ednchina的，现在搬到了sohu；对于LabVIEW的感情很深，去年的NI Days上也有幸见到了真人，非常和蔼的一个LabVIEW老“粉丝”，里面的内容可能会觉得没有阮大侠的那么技术，却更有人情味。</td></tr><tr><td valign="top" width="294"><a href="http://tm.eefocus.com/myspace/blog/indexx.php?bid=43958">http://tm.eefocus.com/myspace/blog/indexx.php?bid=43958</a></td><td valign="top" width="79"></td><td valign="top" width="325">国内某LabVIEW大牛的技术博客，里面对于LabVIEW编程技巧有很多很细致深入的探索。</td></tr><tr><td valign="top" width="294"></td><td valign="top" width="79"></td><td valign="top" width="325"></td></tr><tr><td valign="top" width="294"></td><td valign="top" width="79"></td><td valign="top" width="343"></td></tr></tbody></table>

**Comment by zhangning1639** [comment]
注册lava论坛的时候，填写了名字，163邮箱，密码。点击注册后报"You did not pass the security check"。换了google、夸克、Micro Edge 等浏览器都不行。用了VPN也还是注册不了。请问各位前辈遇到过这个问题吗？有的话，是咋解决的呀？

**Reply by ruanqizhen** [reply]
我搜到了一个类似问题： https://forums.ni.com/t5/BreakPoint/Creating-LAVA-Account/m-p/3909178#M31279 
也许可以过两天再试试看。

**Reply by zhangning1639** [reply]
谢谢，现在NI官方论坛也不给访问了。我搜了下，通过梯子还可以访问。

**Comment by gitliuyawen** [comment]
可怕的是有的程序框图复杂度已经不知道如何优化
